HACKING.md

```diff
@@ -262,35 +262,12 @@ in detail. The project
 examples should be driven by e2e tests. e2e tests can also test external
 components working together.
 
-The end-to-end suite is currently implemented primarily in Bash, but will be
-folded into the extended suite (located in test/extended) over time.
-The extended suite is closer to the upstream Kubernetes e2e suite and
-tests the full behavior of a running system.
+All e2e tests are compiled into the `openshift-tests` binary.
+To build the test binary, run `make build WHAT=cmd/openshift-tests`.
```

Comment:
There's `make build-extended-test` which does just that, eventually `make build-tests` which builds both e2e and integration tests atm.